modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -304,7 +305,8 @@ class CV_EXPORTS_W CascadeClassifier
                                   int minNeighbors = 3, int flags = 0,
                                   Size minSize = Size(),
                                   Size maxSize = Size(),
-                                  bool outputRejectLevels = false );
+                                  bool outputRejectLevels = false,
+                                  int rejectStage = 3);
```

Comment:
i set this to 3 that the previous behaviour before 0bc6092 is restored with the changed logic now